client/src/redux/index.js

```diff
@@ -56,6 +56,7 @@ const initialState = {
   userFetchState: {
     ...defaultFetchState
   },
+  allChallengesInfo: {},
```

Comment:
```suggestion
  allChallengesInfo: {
    challengeEdges: [],
    certificateNodes: [],
  },
```
Since the type has these properties defined.